themes/courses.labster.com/lms/templates/ccx/coach_dashboard.html

```diff
@@ -183,6 +184,21 @@ <h2 class="hd hd-2" id="header-ccx-dashboard">${_("CCX Coach Dashboard")}</h2>
     $errorMessage.hide();
     return true;
   }
+
+  function ccxInviteForm(form) {
+    var emailRegex = /^\s*(?:([A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4})\b[,|\n]*)+$/i;
```

Comment:
@lukyfebrianto this regex will not accept csv with spaces after comma, e.g.
```
vladas@labster.com, user@example.com, another.user@test.org
```
will result in the error message
Is it expected behavior? Maybe add small help text for instructors explaining data format?

Reply:
@TamoshaytisV I already update the regex so it'll handle that case.